app/models/reports/repository.rb

```diff
@@ -147,6 +147,13 @@ def initialize(regions, periods:)
       end
     end
 
+    memoize def ltfu_rates
+      region_period_cached_query(__method__) do |entry|
+        slug, period = entry.slug, entry.period
+        percentage(ltfu[slug][period], denominator(entry.region, period, with_ltfu: true))
```

Comment:
This is not right, need to use the correct denominator here...

Reply:
I would prefer adding a separate method `ltfu_denominator`, or just calling `cumulative_monthly_registrations` directly here. This attempt to make the denominator intelligent wasn't great:

https://github.com/simpledotorg/simple-server/blob/6b4f8bd8fe9e5664b64589f3a165a9d20a054344/app/models/reports/result.rb#L167-L175